packages/devtools_app/integration_test/test_infra/run/_test_app_driver.dart

```diff
@@ -430,15 +430,18 @@ enum TestAppDevice {
   static final _unsupportedTestsForDevice = <TestAppDevice, List<String>>{
     TestAppDevice.flutterTester: [],
     TestAppDevice.flutterChrome: [
-      // TODO(https://github.com/flutter/devtools/issues/5874): Remove once supported on web.
       'eval_and_browse_test.dart',
```

Comment:
should we remove this now?

Reply:
The `eval_and_browse_test` isn't supported on web because it relies on the memory screen (which isn't supported for web). The new test `eval_and_inspect_test` tests for similar functionality on the inspector screen.